hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -558,6 +558,24 @@
       stage: V2
       tests: []
     }
+    {
+      name: chip_rv_dm_end_to_end_debug
```

Comment:
(after offline discussion) perhaps it makes more sense to move this to the rom E2E [testplan](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/rom/data/rom_e2e_testplan.hjson) since we want to run this test with ROM to test the most realistic debug scenario.

Reply:
then we can add a RomE2eTest issue so it ends of up the [GH project dashboard](https://github.com/orgs/lowRISC/projects/13/views/20?groupedBy%5BcolumnId%5D=2972208); note we will want to mark it P2 to indicate we are only planning on running this on FPGA for the time being.

Reply:
As far as I can tell rom_e2e_c_init doesn't have much in common with this test.

I think this better off standalone rather than piggybacking on an existing test (i.e. being part of it, we will reuse existing work to build it). It's a big test as is and combining it with something else could be unwieldy.

Reply:
@GregAC sorry, I pasted the wrong issue, ptal at #14484.